azkaban-web-server/src/main/java/cloudflow/readme.md

```diff
@@ -40,6 +40,68 @@ Sample response body:
   "experimentId" : "none",
   "concurrentOption" : "skip",
   "failureAction" : "FINISH_CURRENTLY_RUNNING",
-  "notifyFailureFirst" : false,
-  "notifyFailureLast" : false
-} ]
\ No newline at end of file
+  "notifyOnFirstFailure" : false,
+  "notifyFailureOnExecutionComplete" : false
+} ]
+
+
+Get job execution details.
+curl -X GET "http://localhost:8081/executions/21024226/jobs/123"
+
+Sample response body:
+{
+  "executionId": "21024226",
+  "startTime": 1580855926989,
+  "endTime": 1580856047132,
+  "status": "FAILED",
+  "attempts": [
+    {
+      "startTime": 1580855926989,
+      "endTime": 1580855957019,
+      "id": 0,
+      "status": "FAILED"
+    },
+    {
+      "startTime": 1580855957032,
+      "endTime": 1580855987060,
+      "id": 1,
+      "status": "FAILED"
+    },
+    {
+      "startTime": 1580855987070,
+      "endTime": 1580856017098,
+      "id": 2,
+      "status": "FAILED"
+    },
+    {
+      "startTime": 1580856017107,
+      "endTime": 1580856047132,
+      "id": 3,
+      "status": "FAILED"
+    }
+  ]
+}
+
+
+Create a flow execution.
+curl -X POST 'http://localhost:8081/executions' \
+-H 'Content-Type: application/json' \
+--data-raw '{
+	"flowId": "2222",
+	"flowVersion": 4,
+	"experimentId": "70",
+	"description": "A test execution.",
+	"failureAction": "finishCurrent",
+	"notifyOnFirstFailure": "true",
+	"notifyFailureOnExecutionComplete": "true",
+	"concurrentOption": "skip",
+	"properties": {
+		"root": {
+			"retries": 3,
+			"failure.emails": ["a@a.com", "b@b.com"],
+			"success.emails": ["c@c.com"]
+		}
+	}
```

Comment:
Do we already use this format for properties somewhere ?
This would need to be documented as part of the API as well. Also wondering if there is a more obvious way of providing rather than using a Map containing Maps.

Reply:
> Do we already use this format for properties somewhere ?
No, today we use `application/x-www-form-urlencoded` as the data format of requests and not `application/json`.
> This would need to be documented as part of the API as well. Also wondering if there is a more obvious way of providing rather than using a Map containing Maps.
I can't think of any but please let me know if you have any suggestions.